src/Microsoft.Azure.WebJobs.ServiceBus/packages.config

```diff
@@ -1,15 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.ServiceBus.EventProcessorHost" version="2.2.5" targetFramework="net45" />
+  <package id="Microsoft.Azure.ServiceBus.EventProcessorHost" version="2.2.6" targetFramework="net45" />
```

Comment:
You also need to update the WebJobs.ServiceBus.nuspec. We have a build task that should be complaining about the mismatched version (or maybe we only added that to the other repos?)